src/io/flutter/editor/WidgetIndentsHighlightingPassFactory.java

```diff
@@ -324,9 +199,32 @@ void runWidgetIndentsPass(EditorEx editor, @NotNull FlutterOutline outline) {
 
   @Override
   public void dispose() {
-    clearListeners();
     FlutterSettings.getInstance().removeListener(settingsListener);
   }
+
+  private class Listener implements FlutterSettings.Listener {
```

Comment:
nit: what is the advantage of defining this as an inner class vs a closure like was done previously?